app/scripts/components/sandbox/analysis-chart/index.js

```diff
@@ -1,18 +1,86 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
-import Chart from '$components/common/chart/block';
+import Chart from '$components/common/chart/analysis/';
+import { AnalysisLegendComponent } from '$components/common/chart/analysis/control';
+import { getColors } from '$components/common/chart/utils';
+
 import { Fold, FoldHeader, FoldTitle } from '$components/common/fold';
 import Constrainer from '$styles/constrainer';
 import { PageMainContent } from '$styles/page';
 import Hug from '$styles/hug';
 
+import timeSeriesData345 from './sample-timeseries-data-345.json';
+import timeSeriesData234 from './sample-timeseries-data-234.json';
+
 const Wrapper = styled.div`
   position: relative;
   grid-column: 1 / -1;
   min-height: 600px;
 `;
+const MainPanel = styled.div`
+  width: 100%;
+  position: relative;
+  grid-column: 1 / -1;
+  background-color: lightgrey;
+  margin-bottom: 50px;
+  padding: 20px;
+`;
+
+// manually collected data from the endpoint below for testing purpose. date ranges from 202203 to 202205
+// https://staging-raster.delta-backend.com/cog/statistics?url="s3://veda-data-store-staging/geoglam/CropMonitor_${date}.tif"1
+
+const dataForChart1 = {
+  timeSeriesData: timeSeriesData345,
```

Comment:
@danielfdsilva as we discussed before, feel free to suggest how the data should be structured.

Reply:
Left a suggestion on https://github.com/NASA-IMPACT/delta-ui/pull/258